geode-web/build.gradle

```diff
@@ -69,5 +69,10 @@ distributedTest.dependsOn(war)
 integrationTest.dependsOn(war)
 
 war {
+  enabled = true
+  rootSpec.exclude("**/*commons-logging-*.jar")
   duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+  // this shouldn't be necessary but if it's not specified we're missing some of the jars
+  // from the runtime classpath
+  classpath configurations.runtimeClasspath
```

Comment:
Similar to Juan's question above - why did we have to why did with add the above lines to the configuration for geode-web?

Reply:
See my comment to Juan.. I do not have answers, only potential solutions based on input here, from people with strong opinions.